Jellyfin.ApiClient/Model/ConnectionResult.cs

```diff
@@ -1,15 +1,13 @@
-﻿using MediaBrowser.Model.Connect;
-using MediaBrowser.Model.Dto;
+﻿using MediaBrowser.Model.Dto;
 using System.Collections.Generic;
 
-namespace Emby.ApiClient.Model
+namespace Jellyfin.ApiClient.Model
 {
     public class ConnectionResult
     {
         public ConnectionState State { get; set; }
         public List<ServerInfo> Servers { get; set; }
         public IApiClient ApiClient { get; set; }
-        public ConnectUser ConnectUser { get; set; }
         public UserDto OfflineUser { get; set; }
```

Comment:
This looks like it's either unnecessary or should be renamed to `User` depending on the usage.

Reply:
I assume you're talking about OfflineUser? VS says 0 references, but the state of server connections/server detection/server switching is not great at the moment. I think I won't touch this until I can fix that functionality, but I do agree that it's probably unnecessary.